last_mile_delivery/cvrptw_benchmark_gehring_homberger.ipynb

```diff
@@ -1,68 +1,129 @@
 {
  "cells": [
   {
-   "cell_type": "code",
-   "execution_count": 16,
-   "id": "afc57ab3",
+   "cell_type": "markdown",
+   "id": "0d500386",
+   "metadata": {},
+   "source": [
+    "# Benchmark Gehring & Homberger\n",
+    "## Capacitated Vehicle Routing Problem with Time Windows (CVRPTW)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "30e63d74",
+   "metadata": {},
+   "source": [
+    "While other notebooks such as [cvrptw_service_team_routing.ipynb](cvrptw_service_team_routing.ipynb) focus on the cuOpt API and high level problem modeling, here we focus on performance.\n",
+    "\n",
+    "cuOpt offers a unique benefit over other solver_settingss, specifically, time to solution.  In addition to achieving world class accuracy, cuOpt also produces these solutions in a time frame that allows for re-optimization in dynamic environments and rapid iteration over possible problem configurations.\n",
```

Comment:
looks like there was a search and replace error here or similar "over other solver_settingss"